metadata-ingestion/src/datahub/configuration/config_loader.py

```diff
@@ -67,12 +68,40 @@ def list_referenced_env_variables(config: dict) -> Set[str]:
     return set([call[1][0] for call in calls])
 
 
+WRITE_TO_FILE_DIRECTIVE_PREFIX = "__DATAHUB_TO_FILE_"
```

Comment:
Is this notion of "directive" a yaml thing or something we've introduce before?

Reply:
it's part of "datahub-flavored yaml", which is something I just made up :)

this change is mainly to make it easy to run UI ingestion - don't expect this to be a common way we do things broadly